src/misc.c

```diff
@@ -347,21 +360,44 @@ int strnzcpylwrn(char *dst, const char *src, int size)
 
 char *strnzcat(char *dst, const char *src, int size)
 {
-	char *dptr = dst;
+	char *dptr;
 
-	if (size) {
-		while (size && *dptr) {
-			size--; dptr++;
-		}
-		if (size)
-			while (--size)
-				if (!(*dptr++ = *src++)) break;
+	if (size < 1) return dst;
```

Comment:
code style, the return should be on new line (several of these as well)

Reply:
Ok, but what about all the prior style breakages in the file (such as the last line in red above, and other returns on same line).   I am pretty sure these were already there due to this code coming from the master project.

I can do this in any way.  Since I am likely going to yank these code changes from this PR, to only add the unit_test logic and nothing else, I can adjust these in a subsequent PR.   I just would like some guidance at that time, trying to be as 'friendly' with the maintainer, trying to get good readable and clean code while at the same time, trying to reduce the master branch merge failures.